src/performance/metrics/PerformanceUtil.cpp

```diff
@@ -1261,13 +1260,17 @@ double PerformanceUtil::getAggregatedLoadAverage(std::string graphId, std::strin
                     "' order by graph_place_sla_performance.place_id,graph_place_sla_performance.elapsed_time LIMIT 2;";
 
                 std::vector<vector<pair<string, string>>> loadAvgResults = perfDb.runSelect(slaLoadQuery);
-                if (loadAvgResults.empty()) {
+
+                if (loadAvgResults.size() >= 2) {
+                    previousLoad = std::atof(loadAvgResults[0][1].second.c_str());
+                    xAxisValue = std::atof(loadAvgResults[0][2].second.c_str());
+                    nextLoad = std::atof(loadAvgResults[1][1].second.c_str());
+                } else if (loadAvgResults.size() == 1) {
+                    //TODO(ASHOK12011234) : Handle the case where there is only one row in the result.
```

Comment:
```suggestion
                    // TODO(ASHOK12011234) : Handle the case where there is only one row in the result.
```

To fix the cpplint failure.